keras_cv/models/csp_darknet.py

```diff
@@ -32,6 +32,21 @@
 from keras_cv.models.__internal__.darknet_utils import SpatialPyramidPoolingBottleneck
 from keras_cv.models.weights import parse_weights
 
+DEPTH_MULTIPLIERS = {
+    "tiny": 0.33,
+    "s": 0.33,
+    "m": 0.67,
+    "l": 1.00,
+    "x": 1.33,
+}
```

Comment:
Are those configs being used anywhere in the codebase?

Reply:
They will be used in YoloX

Reply:
He meant - are the values for these keys used in the model code below.

Reply:
Yep that was my question -- if this is needed in YOLOX, maybe you create a specific darknet with these configs?

Reply:
Ah, makes sense. That's the plan!